web_autoflip_demo/autoflip_demo_single_worker/main.ts

```diff
@@ -0,0 +1,137 @@
+/**
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+interface CropInfo {
```

Comment:
Put a comment explaining the interface, doesn't have to be long. It would also be helpful if you could comment each field.

Reply:
Fixed.

Reply:
Thanks!